### PR TITLE
fix(CroSwap): underlying assets & user positions

### PR DESCRIPTION
**Pull request by ReverendDread**
Made requested changes and fixes.

### CODE REVIEW

File: apps/workshop/projects/CroSwap/modules/CroSwap.ts [hunk 1]
```diff
@@ -72,42 +73,37 @@ export const CroSwap: ModuleDefinitionInterface = {
...
-          token0.decimals,
-        ),
-      );
-      const token1Balance = parseFloat(
-        ethers.utils.formatUnits(
-          reserve1.mulUnsafe(ratioOfTotalSupply).toHexString(),
-          token1.decimals,
-        ),
-      );
+      // // dont continue if no balance
```
Comment (witem) [inline apps/workshop/projects/CroSwap/modules/CroSwap.ts:76]: Please delete all not necessary comments

Comment (ReverendDread) [reply apps/workshop/projects/CroSwap/modules/CroSwap.ts]: removed

**Review by witem — APPROVED** [review]

**Comment by witem** [comment]
Now looks better: 
image

**Comment by witem** [comment]
@ReverendDread all these APRs correct?
image

**Pull request merged**